apl/tabular-operators/union-operator.mdx

```diff
@@ -49,14 +197,93 @@ Combines the datasets, and then filters the data to only include rows where the
 
 [Run in Playground](https://play.axiom.co/axiom-play-qf1k/explorer?initForm=%7B%22apl%22%3A%20%22%5B%27sample-http-logs%27%5D%5Cn%7C%20union%20%5B%27github-issues-event%27%5D%5Cn%7C%20where%20method%20%3D%3D%20%5C%22GET%5C%22%22%7D)
 
-**Aggregating after union**
+### Aggregate after union
 
-Combines the datasets and summarizes the data, counting the occurrences of each combination of `content_type` and `actor`.
+This example combines the datasets and summarizes the data, counting the occurrences of each combination of `content_type` and `actor`.
 
 ```kusto
 ['sample-http-logs']
 | union ['github-pull-request-event']
 | summarize Count = count() by content_type, actor
 ```
 
-[Run in Playground](https://play.axiom.co/axiom-play-qf1k/explorer?initForm=%7B%22apl%22%3A%20%22%5B%27sample-http-logs%27%5D%5Cn%7C%20union%20%5B%27github-pull-request-event%27%5D%5Cn%7C%20summarize%20Count%20%3D%20count%28%29%20by%20content_type%2C%20actor%22%7D)
\ No newline at end of file
+[Run in Playground](https://play.axiom.co/axiom-play-qf1k/explorer?initForm=%7B%22apl%22%3A%20%22%5B%27sample-http-logs%27%5D%5Cn%7C%20union%20%5B%27github-pull-request-event%27%5D%5Cn%7C%20summarize%20Count%20%3D%20count%28%29%20by%20content_type%2C%20actor%22%7D)
```

Comment:
This is weird. I have the opposite where `content-type` is highlighted as problematic. I think at this point the linter acts too unpredictably to try to avoid its issues in the docs.
<img width="461" alt="image" src="https://github.com/user-attachments/assets/e7075457-de48-4d98-a1d9-0388c1eb2bb7">